home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/utils/ApiClientUtil.java

```diff
@@ -25,5 +25,16 @@ public static ApiClient getClient() {
             "No description provided", new HashMap())));
         return client;
     }
+
+    public static ApiClient getUnauthorizedClient() {
+        String applicationExternalPort = System.getProperty("home.application.external.port");
```

Comment:
Let's a bit generalize it. Please move properties to constants. Move common code to a private method.

Also please sync up with @likeRewca. Looks like you have changes in the same place.

Reply:
Synced changes with @likeRewca, moved local variables to common constants, Moved logger registration to private method